docs/new_source.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: page
+title: New Data Source
+permalink: /data/new
+nav_order: 1
+has_children: true
+parent: Data sources
+---
+
+# Contributing A New Data Source
+
+Data contributions are welcome. OSV.dev readily accepts security related records from open source projects, given they are provided in the [OSV format](https://ossf.github.io/osv-schema/). Publishing records to OSV is straightforward: reserve a prefix in the OSV-Schema, and then prepare and publish your data via a Git repository or other supported method. The step by step instructions are as follows:
+
+- [ ] Open an [issue](https://github.com/google/osv.dev/issues) using the 'new data source' template.  
+        
+- [ ] Prepare your data \- refer to the [OSV Schema](https://ossf.github.io/osv-schema/) documentation for information on how to properly format the data so it can be accepted.  
+        
+- [ ] Create a PR to [reserve a prefix in the OSV-Schema](https://ossf.github.io/osv-schema/#id-modified-fields) ([example](https://github.com/ossf/osv-schema/pull/219)). We review the records you start publishing for OSV Schema [correctness](https://github.com/ossf/osv-schema/tree/main/validation) and [quality](https://google.github.io/osv.dev/data_quality.html) as part of reviewing and merging this PR.
```

Comment:
https://github.com/ossf/osv-schema/pull/351 is probably a better example, since there's some extra bits that need to be edited.